spec/Overview.html

```diff
@@ -7135,14 +7126,15 @@ <h5>Sign</h5>
                       <li>
                         <p>
                           Perform the ECDSA signing process, as specified in [[RFC6090]],
-                          Section 5.4, with |M| as the message, using |params| as the
-                          EC domain parameters, and with |d| as the private key.
+                          Section 5.4.2, with |m| as the message,
+                          and |z| as the private key,
+                          using |params| as the EC domain parameters.
                         </p>
                       </li>
                       <li>
                         <p>
-                        Let |r| and |s| be the pair of integers resulting from
-                        performing the ECDSA signing process.
+                          Let |s1| and |s2| be the pair of integers resulting from
```

Comment:
Why change this? The canonical names for the components of an ECDSA signature are `r` and `s`.